guidelines/dxp/extracting_js_from_jsps.md

```diff
@@ -0,0 +1,41 @@
+# Extracting JS from JSPs
+
+As we're modernizing our codebase across DXP we are starting to encounter cases where we are touching JS code inside JSPs and often times we have to remind ourselves how best to tackle that.
+
+We have two options when extracting JS from JSPs:
+
+-   `<liferay-frontend:component>` - good ol' JS that doesn't render UI
```

Comment:
I would use `old` here, per the same reason we don't use abbreviations in source code: some people's English knowledge may be reduced and they may not understand what it means.

Reply:
So, keep it boring :joy:

Reply:
True, my inner Tolkien came through there

Reply:
![incredible](https://media.giphy.com/media/ckvuK4mlUuohi/giphy.gif)